cantabular/gql/data.go

```diff
@@ -26,7 +26,7 @@ type Node struct {
 	Label      string      `json:"label"`
 	Categories Categories  `json:"categories"`
 	MapFrom    []Variables `json:"mapFrom"`
-	FilterOnly string      `json:"filterOnly"`
+	FilterOnly string      `json:"filterOnly,omitempty"`
```

Comment:
this is just for my own learning: why are some of these values in `gql` package and others are in `cantabular` package (`contract.go`)? Both end up being part of the client's "contract" don't they?

Reply:
I tried to separate the 'common' types from the contract, but maybe you are right and these types can be part of the contract.

Reply:
Maybe I will leave it as it is, as this can be used by multiple different generic graphql respones. But feel free to change it in the future.